lua/nvim-tree/actions/fs/rename-file.lua

```diff
@@ -53,7 +53,9 @@ function M.fn(default_modifier)
 
     -- support for only specific modifiers have been implemented
     if not ALLOWED_MODIFIERS[modifier] then
-      return notify.warn("Modifier " .. vim.inspect(modifier) .. " is not in allowed list : " .. table.concat(ALLOWED_MODIFIERS, ","))
+      return notify.warn(
+        string.format("Modifier %s is not in allowed list: %s", vim.inspect(modifier), table.concat(ALLOWED_MODIFIERS, ","))
```

Comment:
As @gegoune said in #2508, this is a change outside of the scope of your PR. Using `string.format` here is arguably better, but it should be done in a separate refactoring.

Reply:
I am sorry. I understood gegoune's words to fix that as well.

Reply:
Well, now that I think about it, I'm not sure if the "same here" is about the unrelated style changes or the use of `string.format`... Let's wait for his feedback.

Reply:
Okay and I finally understand your FYI. Thank you for your kindness.

Reply:
Apologies if I wasn't clear enough. My fault. 

Regarding unrelated changes: I meant it for formatting changes toward the top of file. 

`string.format` is what we should be using going forward but as @Akmadan23 said existing code should not be changed as part of that pr to make it self contained. We can replace other string concatenations across code base in separate pr (I can do it). 

Hope that clears things up. Thanks for your pr!